saleor/graphql/checkout/mutations.py

```diff
@@ -351,6 +371,7 @@ def perform_mutation(cls, _root, info, checkout_id, lines, replace=False):
         )
 
         variant_ids = [line.get("variant_id") for line in lines]
+        info.context.channel_slug = checkout.channel.slug
```

Comment:
Why do we need this? This value comes from middleware and IMO it should be always set by the middleware, otherwise we'll end up with magic which is really difficult to debug.